src/control/cmd/dmg/README.md

```diff
@@ -39,26 +62,61 @@ domains which map to individual subcommands.
 For exhaustive subcommand listings please see output of
 `dmg [<subcommand>] --help`.
 
+### Config
+
+Provides capability to automatically generate recommended server
+config file for a given set of hosts.
+Implementation in `auto.go`.
+
+### Container
+
+Provides capability to change the owner of a DAOS container.
+Implementation in `cont.go`.
+
+### Firmware
+
+Firmware related capabilities are selectively compiled and may not
+be present in all builds. Provided capability to query and update
+firmware on NVMe and PMem devices.
+Implementation in `firmware.go` and stub in `firmware_disabled.go`.
```

Comment:
How about "Implementation with firmware enabled in `firmware.go`. When disabled: `firmware_disabled.go`"